### PR TITLE
stride-aware operations.

### PR DESCRIPTION
**Pull request by poilvert**

### CODE REVIEW

File: sthor/model/slm_new.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
+"""Sequential Layered Model"""
```
Comment (npinto) [inline sthor/model/slm_new.py]: this file has nothing to do with the stride PR

File: sthor/model/slm_nugget.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import numpy as np
```
Comment (npinto) [inline sthor/model/slm_nugget.py]: this file has nothing to do with the stride PR

Comment (poilvert) [inline sthor/model/slm_nugget.py]: indeed. removed the files that were wrongly here.

On Wed, Apr 11, 2012 at 3:02 PM, Nicolas Pinto <
reply@reply.github.com

> wrote:
> 
> > @@ -0,0 +1,28 @@
> > +import numpy as np
> 
> this file has nothing to do with the stride PR
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/nsf-ri-ubicv/sthor/pull/8/files#r672582

## 

Nicolas POILVERT, PhD
Post-doctoral Fellow
Rowland Institute at Harvard

**Comment by npinto** [comment]
Looks good.

**Comment by poilvert** [comment]
awesome!

On Wed, Apr 11, 2012 at 3:16 PM, Nicolas Pinto <
reply@reply.github.com

> wrote:
> 
> Looks good.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/nsf-ri-ubicv/sthor/pull/8#issuecomment-5076653

## 

Nicolas POILVERT, PhD
Post-doctoral Fellow
Rowland Institute at Harvard

**Pull request merged**